ui/apps/platform/src/types/featureFlag.ts

```diff
@@ -4,6 +4,7 @@
 export type FeatureFlagEnvVar =
     | 'ROX_ACTIVE_VULN_MGMT'
     | 'ROX_CLUSTERS_PAGE_MIGRATION_UI'
+    | 'ROX_CUSTOMIZABLE_PLATFORM_COMPONENTS'
```

Comment:
@charmik-redhat I may need to utilize the feature flag soon. It might be worthwhile to do the feature flag addition in a separate PR so we can get that in sooner.

Reply:
Yeah you can create a separate PR for the feature flag